README.md

```diff
@@ -1,2 +1,91 @@
-# inferno-backend
+# Inferno backend
+
 The backend of the Inferno Rewards project. It's intended to determine the awarded addresses and to distribute the weekly rewards to them.
+
+## Setup
+
+Install dependencies
+
+ - [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)
+
+Then run:
+
+```sh
+$ nvm install && nvm use
+$ npm add -g yarn
+$ yarn && yarn build
+```
+
+At this point you will have a directory at `./lib`, there will be the compiled JS files.
+
+## Cache
+
+The folder `./cache` contains the immutable cached blocks. Every time you run the `aggregate` command, the non-cached blocks will be added there.
+
+## Run the aggregation proccess
+
+You can call the built script by running:
+```sh
+$ yarn node ./lib/index.js aggregate
+```
+to aggregate data since the previously aggregated block (defaults to the height specified at `./daily_output.base.json`) until the current height.
```

Comment:
Explain that it defaults to the latest output or the base in case the output doesn't exist

Reply:
b6f28a9991406ba698b6b8e24886219ae56d62d8